src/components/Restoration/YourCompanySummary.vue

```diff
@@ -4,15 +4,39 @@
     <div class="summary-header px-4 mb-2 rounded-t">
       <v-row no-gutters>
         <v-col cols="9">
-          <img class="header-icon" src="@/assets/images/currency-usd-circle.svg">
+          <v-icon class="header-icon">mdi-domain</v-icon>
           <label class="summary-title pl-2">Your Company</label>
         </v-col>
       </v-row>
     </div>
-    <!-- Business Name -->
+
     <template>
       <v-divider class="mx-4" />
-      <div class="section-container business-name-summary">
+
+      <!-- Business Name -->
+      <template v-if="hasBusinessNameChanged">
+        <div class="section-container business-name-summary">
+          <v-row no-gutters>
+            <v-col cols="3">
+              <label><strong>Company Name</strong></label>
+            </v-col>
+
+            <v-col cols="8" class="mt-n1">
+              <div class="company-name font-weight-bold text-uppercase">{{ getCompanyName }}</div>
+              <div class="company-name mt-2">{{ getNameRequest.nrNumber }}</div>
+            </v-col>
+          </v-row>
+        </div>
+      </template>
+
+      <!-- Name Translation -->
+      <template v-if="haveNameTranslationsChanged">
+        <div class="section-container name-translation-summary">
+          <NameTranslation :isSummaryMode="true" />
+        </div>
+      </template>
+
+      <div class="section-container">
         <v-row no-gutters>
```

Comment:
Added Business name and Name translation as they are shown in one block as per UXpin